tools/cli/new-template.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const { ALLOWED_INTEGRATION_TYPES, INTEGRATION_PLURAL } = require("./constants");
+const { getAbsolutePath, extractContentFromFile, replaceContentInFile } = require("./utils");
+const fs = require("fs");
+const path = require("path");
+
+const args = process.argv.slice(2);
+
+if (!args[0] || !args[1]) {
+    console.error("Error: Please provide the integration type and integration ID as arguments.");
+    process.exit(1);
+}
+
+const integrationTypeSin = args[0];
+const integrationTypePlu = INTEGRATION_PLURAL[integrationTypeSin];
```

Comment:
The variable names with `sin` and `plu` harm readability. Shall we update to have better variable names?